aspnetcore/host-and-deploy/docker/building-net-docker-images.md

```diff
@@ -164,7 +164,7 @@ In some scenarios, you might want to deploy an app to a container by copying to
 To use the manually published application within a Docker container, create a new Dockerfile and use the `docker build .` command to build the container.
 
 ```console
-FROM mcr.microsoft.com/dotnet/core/aspnet:2.2 AS runtime
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.0 AS runtime
```

Comment:
We should be using the monikers and adding support for multiple versions 
`::: moniker range=">= aspnetcore-3.0"`

@Rick-Anderson can you point to the documentation on how to add these?